hack/rp-dev/rp_funcs.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash -e
+######## Helper file with the functions to fully deploy RP dev either locally or using Azure DevOps Pipelines ########
+
+source hack/devtools/rp_dev_helper.sh
+
+setup_rp_config() {
+  # Check if exactly non-empty two arguments are provided
+  if [[ $# -ne 2 ]]; then
+    echo "Error: $0 <AZURE_PREFIX> <SKIP_DEPLOYMENTS>. Please try again"
+    exit 1
+  fi
+
+  # TODO - check if needed
+  export USER=dummy
+  # Export the SECRET_SA_ACCOUNT_NAME environment variable and run make secrets
+  az account set -s fe16a035-e540-4ab7-80d9-373fa9a3d6ae
+  export SECRET_SA_ACCOUNT_NAME=rharosecretsdev && make secrets
+
+  source hack/devtools/rp-dev-helper.sh && verify_downloading_secrets
+
+  export LOCATION=eastus
```

Comment:
RP full dev is created at location `eastus`, and it can be propagated as a variable (e.g., _AZURE_PREFIX_/_SKIP_DEPLOYMENTS_).

Reply:
Done in the last commit